Core/tests/ExpressionParser2.cpp

```diff
@@ -340,6 +340,20 @@ TEST_CASE("ExpressionParser2", "[common][events]") {
               "Cannot find an expression with this name: Idontexist\n"
               "Double check that you've not made any typo in the name.");
     }
+    {
+      auto node = parser.ParseExpression("🅸🅳🅾🅽🆃🅴🆇🅸🆂🆃😄(\"hello\"");
```

Comment:
The parser was already designed to support Unicode in identifiers, I just added a few tests.
Let me know @D8H if you think there are some risks or things to cover with tests.